Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/JunctionCenteringTest.java

```diff
@@ -13,22 +16,28 @@ public class JunctionCenteringTest extends BaseAutonomous {
     public void runOpMode() throws InterruptedException {
         initialize();
 
-        grabberCameraPipeline.setRanges(Constants.LOWER_YELLOW, Constants.UPPER_YELLOW);
+        driveGrabber(Constants.GRABBER_CLOSE_POSITION);
 
-        grabberCamera.openCameraDeviceAsync(new OpenCvCamera.AsyncCameraOpenListener() {
-            @Override
-            public void onOpened() {
-                grabberCamera.startStreaming(Constants.CAMERA_X, Constants.CAMERA_Y, OpenCvCameraRotation.UPRIGHT);
-            }
+        grabberCameraPipeline.setRanges(Constants.LOWER_BLACK, Constants.UPPER_BLACK);
 
-            @Override
-            public void onError(int errorCode) {}
-        });
+        startCameraWithPipeline(grabberCameraPipeline, grabberCamera, Constants.CAMERA_X, Constants.CAMERA_Y);
 
-        grabberCamera.setPipeline(grabberCameraPipeline);
+        telemetry.addData("x", grabberCameraPipeline.xPosition-Constants.CAMERA_CENTER_X);
+        telemetry.addData("y", Constants.CAMERA_CENTER_Y-grabberCameraPipeline.yPosition);
+        telemetry.addData("xPower", junctionTopPixelsMotorPower(grabberCameraPipeline.xPosition-Constants.CAMERA_CENTER_X));
+        telemetry.addData("yPower", junctionTopPixelsMotorPower(Constants.CAMERA_CENTER_Y-grabberCameraPipeline.yPosition));
+        telemetry.update();
 
         waitForStart();
 
         centerJunctionTop(grabberCameraPipeline);
+
+        while (opModeIsActive()){
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace
```suggestion
        while (opModeIsActive()) {
```

Reply:
Added